pom.xml

```diff
@@ -95,36 +107,31 @@
             <artifactId>httpcore</artifactId>
             <version>4.4.14</version>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci</groupId>
-            <artifactId>trilead-ssh2</artifactId>
-            <version>build-217-jenkins-27</version>
-        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.modules</groupId>
             <artifactId>instance-identity</artifactId>
-            <version>2.2</version>
+            <!--<version>managed by bom</version>-->
```

Comment:
I thought that something obvious like this to show where the version is coming from would reduce confusion in the future, but happy to remove these comments entirely too.